src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter((x) => x.sex === 'm');
+
+  if (century) {
+    men = men.filter(onePeople =>
+      century === Math.ceil(onePeople.died / 100)
+    );
+  }
+
+  const agesInArray = men.map(x => x.died - x.born);
+  const sumArray = agesInArray.reduce((sum, index) => sum + index);
```

Comment:
you can combine two methods
```suggestion
  const sumOfAges = men.reduce((sum, man) => sum + (man.died - man.born), 0);
```

also about naming:
sumArray is a bad example, it's clear it's an array, better explain the sum of what you have